test/Object/ObjectSpread_JIT.js

```diff
@@ -0,0 +1,47 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Object Spread JIT unit tests
+
+if (this.WScript && this.WScript.LoadScriptFile) { // Check for running in ch
+    this.WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");

```

Comment:
I think UnitTestFramework + JIT aren't going to play nicely together, since the test framework itself will get impact jitting. If you look at inlining tests, for example, they are all bare JS files looking for very specific behaviors. I think you will want something like that here, though I am not sure what the best place and names for such tests would be.

Reply:
Talked to Rajat, UnitTestFramework + JIT should be fine.